nodejs/device/src/thiefSettings.d.ts

```diff
@@ -0,0 +1,14 @@
+export type thiefSettings = {
```

Comment:
>t [](start = 12, length = 1)

I would capitalize the first letter of this type (and interface) name.  Here and everywhere.